Makefile

```diff
@@ -1,231 +1,148 @@
+	mkdir -p debian
+	touch debian/control
+
+	$(eval DEPENDS := $(shell find _build/default/rel/vernemq -type f -name "*.so" | xargs dpkg-shlibdeps -O | cut -c16- | sed "s/,/\" --depends \"/g"))
+	fpm -s dir -t deb -v "$(GIT_VERSION)" \
+		--force \
+		--name vernemq \
+		--epoch $(NOW) \
+		--license "Apache 2.0" \
+		--url "https://vernemq.com" \
+		--vendor "Octavo Labs AG" \
+		--maintainer "<info@vernemq.com>" \
+		--description "VerneMQ is a MQTT message broker" \
+		--depends logrotate \
+		--depends sudo \
+		--depends adduser \
+		--depends "$(DEPENDS)" \
+		--deb-user vernemq \
+		--deb-group vernemq \
+		--deb-changelog ./changelog.md \
+		--deb-no-default-config-files \
+		--deb-systemd files/vernemq.service \
+		--after-install files/deb-vernemq.postinst \
+		--config-files /etc/vernemq/vernemq.conf \
+		_build/default/rel/vernemq/bin/vernemq=/usr/bin/vernemq \
+		_build/default/rel/vernemq/bin/vmq-admin=/usr/bin/vmq-admin \
+		_build/default/rel/vernemq/data=/var/lib/vernemq/ \
+		_build/default/rel/vernemq/etc/=/etc/vernemq/ \
+		_build/default/rel/vernemq/bin/=/usr/lib/vernemq/bin/ \
+		_build/default/rel/vernemq/lib=/usr/lib/vernemq/ \
+		_build/default/rel/vernemq/releases=/usr/lib/vernemq/ \
+		_build/default/rel/vernemq/erts-$(shell erl -eval 'erlang:display(erlang:system_info(version)), halt().'  -noshell)=/usr/lib/vernemq/ \
+		_build/default/rel/vernemq/share/=/usr/share/vernemq/ \
+		_build/default/rel/vernemq/log/=/var/log/vernemq/
+
+rpm: OVERLAY_VARS=vars/rpm_vars.config
+rpm: pkg_rel pkg_strip
+
+	fpm -s dir -t rpm -v "$(GIT_VERSION)" \
+		--force \
+		--name vernemq \
+		--epoch $(NOW) \
+		--license "Apache 2.0" \
+		--url "https://vernemq.com" \
+		--vendor "Octavo Labs AG" \
+		--maintainer "<info@vernemq.com>" \
+		--description "VerneMQ is a MQTT message broker" \
+		--depends logrotate \
+		--depends sudo \
+		--rpm-user vernemq \
+		--rpm-group vernemq \
+		--rpm-autoreqprov \
+		--rpm-compression xz \
+		--before-install files/rpm-vernemq.preinst \
+		--after-install files/rpm-vernemq.postinst \
+		--config-files /etc/vernemq/vernemq.conf \
+		files/vernemq.service=/etc/systemd/system/vernemq.service \
+		_build/default/rel/vernemq/bin/vernemq=/usr/sbin/vernemq \
+		_build/default/rel/vernemq/bin/vmq-admin=/usr/sbin/vmq-admin \
+		_build/default/rel/vernemq/data=/var/lib/vernemq/ \
+		_build/default/rel/vernemq/etc/=/etc/vernemq/ \
+		_build/default/rel/vernemq/bin/=/usr/lib64/vernemq/bin/ \
+		_build/default/rel/vernemq/lib=/usr/lib64/vernemq/ \
+		_build/default/rel/vernemq/releases=/usr/lib64/vernemq/ \
+		_build/default/rel/vernemq/erts-$(shell erl -eval 'erlang:display(erlang:system_info(version)), halt().'  -noshell)=/usr/lib64/vernemq/ \
+		_build/default/rel/vernemq/share/=/usr/share/vernemq/ \
+		_build/default/rel/vernemq/log/=/var/log/vernemq/
+
+pkg_clean:
+	rm -rf _build/default/rel
 
 ##
 ## Developer targets
 ##
 ##  devN - Make a dev build for node N
 dev% :
```

Comment:
The dev targets doesn't seem to work.

I tried doing `make dev1 dev2 dev3` and then when starting I get:

```
$ _build/dev1/rel/vernemq/bin/vernemq start
!!!!
!!!! WARNING: ulimit -n is 1024; 65536 is the recommended minimum.
!!!!
vmq_cluster_node_sup
vernemq failed to start within 15 seconds,
see the output of 'vernemq console' for more information.
If you want to wait longer, set the environment variable
WAIT_FOR_ERLANG to the number of seconds to wait.
```

And it doesn't look like vernemq was started.